transitions/extensions/diagrams.py

```diff
@@ -74,7 +74,23 @@ def _add_edges(self, events, container):
                 src = transitions[0]
                 for t in transitions[1]:
                     dst = t.dest
-                    container.add_edge(src, dst, label=label)
+                    lbl = self._transition_label(label, t)
+                    container.add_edge(src, dst, label=lbl)
+
+    @staticmethod
+    def edge_label_from_transition_label(label):
```

Comment:
Is there a reason to have this as a static method in this class? I don't see it invoked from anywhere other than the tests right now--in which case it should probably be moved into the test class.